### ISSUE TITLE
fix all warnings

### ISSUE THREAD

**Issue by futurepaul** [emeceve/loquaz#17]
would love to jam through and fix through all the warnings (a lot of them are `await` that need to be unwrapped or return errors or something). would greatly improve the dev experience for me personally :)

**Comment by 86667** [comment]
Is this repo stilll active? I will pick this up if so

**Comment by emeceve** [comment]
@86667 yes, but we are changing out frontend framework from druid to tauri. Take a look at the `main-tauri` branch
